usaspending_api/api_contracts/contracts/v2/reporting/agencies/agency_code/publish_dates.md

```diff
@@ -0,0 +1,79 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agency Reporting Publish Dates [/api/v2/reporting/agencies/{agency_code}/publish_dates?{fiscal_year,fiscal_period,search,page,limit,order,sort}]
+
+This endpoint is used to power USAspending.gov's about the data submission history modal. This data can be used to better understand the ways agencies submit data.
+
+## GET
+
+This endpoint returns an overview of government agencies submission data.
+
++ Parameters
+    + `agency_code`: `020` (required, string)
+        The specific agency code.
+    + `fiscal_year`: 2020 (required, number)
+        The fiscal year.
+    + `fiscal_period`: 10 (required, number)
+        The fiscal period.
+        + `page` (optional, number)
+        The page of results to return based on the limit.
+        + Default: 1
+    + `limit` (optional, number)
+        The number of results to include per page.
+        + Default: 10
+    + `order` (optional, enum[string])
+        The direction (`asc` or `desc`) that the `sort` field will be sorted in.
+        + Default: `desc`
+        + Members
+            + `asc`
+            + `desc`
+    + `sort` (optional, enum[string])
+        A data field that will be used to sort the response array.
+        + Default: `publication_date`
+        + Members
+            + `publication_date`
+            + `certification_date`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + `results` (required, array[AgencyData], fixed-type)
```

Comment:
`AgencyData` is not defined in the Data Structures

Reply:
fixed